src/modules/feed/components/Transaction/Transaction.js

```diff
@@ -1,7 +1,10 @@
 import { h } from "preact";
 import { Card } from "semantic-ui-react";
-import { LikeButton } from "./LikeButton";
-import Header from "./Header/Header";
+import LikeButton from "./LikeButton";
+import Header from "./Header";
+import settings from "../../../../config/settings";
```

Comment:
For readability, would it help to link to it from the root? (using a webpack resolve alias?)